cmd/minikube/cmd/start_flags.go

```diff
@@ -636,7 +643,7 @@ func generateNewConfigFromFlags(cmd *cobra.Command, k8sVersion string, rtime str
 			APIServerName:          viper.GetString(apiServerName),
 			APIServerNames:         apiServerNames,
 			APIServerIPs:           apiServerIPs,
-			DNSDomain:              viper.GetString(dnsDomain),
+			DNSDomain:              defaultClusterDNSDomain,
```

Comment:
The rename will make this much more clear and correct:

```go
DNSDomain: dnsDomain
```

Reply:
dnsDomain is a const. I didn't want to shadow the name.